_docs/extensibility/adding-mappings-source.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: Mappings Source Extensions
+meta_title: Mappings Source Extensions
+description: Adding additional source of loading mappings via extensions.
+---
+
+Additional source to load the stub mappings can be configured by implementing `MappingsSourceExtension`.
+
+```java
+public class DummyMappingsSourceExtension extends MappingsSourceExtension {
+  @Override
+  public String getName() {
+    return null; // Return the name of extension
+  }
+
+  @Override
+  public void loadMappingsInto(StubMappings stubMappings) {
+    {
+        // implementation to load the mappings
+        // mappings can be loaded from any source like git repo, database, file storage, stc
+    }
+  }
+}
+```
+
+Registering the extension with wiremock.
+
+```java
+public class Main {
```

Comment:
Suggest removing all the main class stuff and just show newing up `WireMockServer`. This is how the other examples are done.

Reply:
All 5 feedback point incorporated